### PR TITLE
fix: cors when download an image

### LINKED ISSUE #4772: CORS errors on download
### Step to reproduce
- on beta/preview
- own nft
- go to GalleryItem detail
- in more action button click on "Download"

Capture d’écran 2023-01-19 à 3 05 49 PM

```
GET | https://image.w.kodadot.xyz/ipfs/bafkreie7cnvyrxo3flxdsnw6lxlrjnh7hs75taqbox7omahxvekg74pmf4
```

+ wrong item fetch

### PR DESCRIPTION
**Pull request by preschian**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [x] Bugfix
- [ ] Feature
- [ ] Refactoring

### What's new?

- [x] PR closes #4772

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [ ] I've posted a screenshot of demonstrated change in this PR

### Optional

- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=DY4SQF2iD456tH89aQtz5wv1EV3BbSW8wKKuMcwbmXaj1pM)

### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [ ] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

### CODE REVIEW

File: components/gallery/GalleryItemMoreActionBtn.vue [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
     <template #items>
       <NeoDropdownItem
-        v-if="currentGalleryItemImage.mimeType.includes('image')"
+        v-if="mimeType.includes('image') && ipfsImage && name"
```
Comment (Jarsen136) [inline components/gallery/GalleryItemMoreActionBtn.vue:7]: I think the `name` is not necessary because there is a default name for the image.

**Review by Jarsen136 — COMMENTED** [review]
WFM

**Review by roiLeo — APPROVED** [review]
🧪 tested
✅ wfm

**Review by Jarsen136 — APPROVED** [review]
code lgtm

File: nuxt.config.js [hunk 2]
```diff
@@ -244,7 +244,7 @@ export default defineNuxtConfig({
       theme_color: '#181717',
     },
     workbox: {
-      enabled: true,
+      // enabled: true, // enable this to use workbox in localhost
```
[the remark this answers is not in the document]
Comment (roiLeo) [reply nuxt.config.js:247]: can't you use NODE_ENV or something else instead of removing it if this line is needed even on local env

File: components/gallery/GalleryItemMoreActionBtn.vue [hunk 3]
```diff
@@ -14,13 +14,16 @@
 
 <script setup lang="ts">
 import { NeoButton, NeoDropdown, NeoDropdownItem } from '@kodadot1/brick'
-import { downloadImage } from '~/utils/download'
+import { downloadImage } from '@/utils/download'
+import { ipfsToCf } from '@/utils/ipfs'
 
-const { $store } = useNuxtApp()
-const currentGalleryItemImage = $store.getters['history/getCurrentlyViewedItem']
```
[the remark this answers is not in the document]
Comment (roiLeo) [reply components/gallery/GalleryItemMoreActionBtn.vue:20]: Nah we good, I'll reintroduce these lines

**Comment by yangwao** [comment]
pay 50 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $50 @ 33.68 USD/KSM ~ 1.485 $KSM
🧗 [DY4SQF2iD456tH89aQtz5wv1EV3BbSW8wKKuMcwbmXaj1pM](https://kusama.subscan.io/account/DY4SQF2iD456tH89aQtz5wv1EV3BbSW8wKKuMcwbmXaj1pM)
🔗 [0x03bdf6d65a5e0938b1ef847b686873f0ebbcc8b6c2bbfcf5c4fa791ee4eae9e6](https://kusama.subscan.io/extrinsic/0x03bdf6d65a5e0938b1ef847b686873f0ebbcc8b6c2bbfcf5c4fa791ee4eae9e6)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)
